src/components/timeline/TimelineViewControls.svelte

```diff
@@ -160,6 +179,17 @@
   />
 {/if}
 
+<button
+  class="st-button icon"
+  on:click={onCopyViewportURL}
+  use:tooltip={{
+    content: `Copy URL of current viewport to clipboard.${$viewIsModified ? ' View has unsaved changes.' : ''}`,
```

Comment:
Should we be communicating that this is also potentially copying your selected sim dataset and/or selected directive?

Reply:
I think the sim dataset warrants explicit call out as there cues are less obvious and it's easy to assume the latest dataset is shown